core/storage/trie/polkadot_trie/polkadot_trie_cursor.cpp

```diff
@@ -19,6 +19,8 @@ OUTCOME_CPP_DEFINE_CATEGORY(kagome::storage::trie,
              "end or not calling next() after initialization";
     case E::NULL_ROOT:
       return "the root of the supplied trie is null";
+    case E::NOT_FOUND:
+      return "the seeked value is not found";
```

Comment:
```suggestion
      return "the value being looked for was not found";
```
seek - sought - sought

Reply:
I will replace it with sought then.